pyro/contrib/glmm/__init__.py

```diff
@@ -0,0 +1,35 @@
+"""
```

Comment:
You need some basic tests for this module- check same log probs obtained from this and truth models